devops/.trivyignore.yaml

```diff
@@ -0,0 +1,5 @@
+misconfigurations:
+  - id: AVD-DS-0001
```

Comment:
@AlexeySachkov did you see this issue? I don't think I observed it previously when creating this PR. But now it's here, don't know why, likely just magic.

@aelovikov-intel should we switch containers to use a specific hash instead of "latest" or the justification below is fair enough? (see https://avd.aquasec.com/misconfig/dockerfile/general/avd-ds-0001/)

Reply:
No idea. @sarnex , @uditagarwal97 , any thoughts?

Reply:
I can think of two events when our images are rebuilt:
- GPU driver update. This event is triggered/controlled by us
- Nightly build happens. This event is also triggered/controlled by us and we _do_ want to use the latest image there to test the actual latest version of the project and not something old

Both of those seems to be "controlled" updates. And I also assume that our baseline images are pretty stable and don't change much (i.e. at all), Therefore, ignoring this is fine by me

Reply:
agree with alexey